core/invoker/src/main/scala/whisk/core/container/package.scala

```diff
@@ -83,7 +83,9 @@ package object container {
         override def toString() = s"$host:$port"
     }
 
-    sealed abstract class ContainerIdentifier(val id: String)
```

Comment:
I would prefer that we avoided that (to make sure we never use `String`s when something else makes more semantic sense. It's on me that I didn't catch all the required `.name`, though.